<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seller</class>
 <widget class="QDialog" name="seller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Аккаунт Продавца</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>921</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Каталог товаров</string>
    </attribute>
    <attribute name="toolTip">
     <string>Редактирование товаров</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>701</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Здесь вся та же фигня, что и с заказами.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Товары и редактирование в одной вкладке.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Название товара, картинка(всё ещё не уверена), описание(возможно тут будет &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;указываться размер и цвет), наличие на складе.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Товар закончился? Оповести об этом продавца, подсвети товар красным цветом.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Товар не берут уже несколько месяцев? Он лежит и пылится на складе?&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Пусть он будет серым, пусть продавец задумается о рекламе.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Какой то товар берут очень часто? Больше 40% от общего кол-ва продаж? &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Он достоин фиолетового цвета!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;----------------------------------------------------------------------&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;возможно эта информация должна быть в статистике, &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;но писался текст с конца и нормально мысль сформировалась только сейчас&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_products">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>521</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>225</number>
     </attribute>
     <column>
      <property name="text">
       <string>Название</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Описание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Наличие</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>530</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить товар</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>530</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить таблицу</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>530</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить товар</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Заказы</string>
    </attribute>
    <attribute name="toolTip">
     <string>Заказы</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>711</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Каким то образом нужно отобразить людей и их заказы.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Насчет БД: пусть все товары хранятся в одном файле, &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;просто у них будет доп. поле с именем продавца. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Ещё один файл с паролями и ролями. Файл с заказами. Что то вроде Товар : Продавец : Покупатель.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Чтобы отобразить в табличке мы с помощью поля &amp;quot;Продавец&amp;quot; узнаем,&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;у какого продавца товар заказан и у кого должен отображаться.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Поле &amp;quot;Товар&amp;quot; поможет вытащить инфу из базы с товарами.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Поле &amp;quot;Покупатель&amp;quot; для отображения заказа у покупателя и для отображения имени у продавца.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Над оплатой заказов не заморачивайся. Сделаем вид(или кнопку), что товар оплачен.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_orders">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>521</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>180</number>
     </attribute>
     <column>
      <property name="text">
       <string>Название товара</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ник покупателя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Подтвердить оплату</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Удалить заказ</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>530</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Статистика товаров</string>
    </attribute>
    <attribute name="toolTip">
     <string>Здесь отображается статистика товаров за все время</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>521</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="update_btn">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>530</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>789</x>
     <y>10</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти из аккаунта</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
